services/smtp.py

```diff
@@ -0,0 +1,44 @@
+import smtplib
+import ssl
+from functools import cached_property
+
+from shared.config import get_config
+
+from helpers.email import Email
+
+_smtp_service = None
+
+
+def get_smtp_service():
+    if len(get_config("services", "smtp", default={})) == 0:
+        return None
+    return _get_cached_smtp_service()
+
+
+def _get_cached_smtp_service():
+    global _smtp_service
+    if _smtp_service is None:
+        _smtp_service = SMTPService()
+    return _smtp_service
+
+
+class SMTPService:
+    def __init__(self):
+        self.host = get_config("services", "smtp", "host", default="mailhog")
```

Comment:
Does the connection work if I don't specify username and password?

Reply:
Yes